src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -263,6 +263,71 @@ events[counter] is CloudEvent
         }
     }
 
+#nullable enable
+    public override async Task<BulkPublishResponse<byte[]>> PublishBulkByteEventAsync(
+        string pubsubName,
+        string topicName,
+        IReadOnlyList<ReadOnlyMemory<byte>> events,
+        string dataContentType = "application/json",
+        Dictionary<string, string>? metadata = null,
+        CancellationToken cancellationToken = default)
+    {
+        var request = new Autogenerated.BulkPublishRequest
```

Comment:
Is there a reason we cannot refactor and use the existing generic `MakeBulkPublishRequest` here instead of duplicating the method for this additional type?